examples/AvroGeneric/Program.cs

```diff
@@ -98,13 +98,13 @@ static async Task Main(string[] args)
             {
                 Console.WriteLine($"{producer.Name} producing on {topicName}. Enter user names, q to exit.");
 
-                long i = 1;
+                long i = 0;
```

Comment:
future note: generally best to avoid doing unrelated changes - to keep history/blame as useful as possible. but that's fine, just noting for the future.

Reply:
Ok